### PR TITLE
[tw] Localize 'Continuous Delivery'

### LINKED ISSUE #2168: [tw] `Continuous Delivery (CD)` for `Traditional Chinese`
#### Why this term needs to be updated? (briefly) 
- Localize Continuous Delivery (CD) in Traditional Chinese
- https://glossary.cncf.io/continuous-delivery/

### PR DESCRIPTION
**Pull request by tico88612**
### Describe your changes

Origin content from https://glossary.cncf.io/continuous-delivery/

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2168 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/zh-tw/continuous-delivery.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: 持續交付 (CD)
+status: Completed
+category: 概念
+tags: ["應用程式", "方法論", ""]
+---
+
+## 是什麼 {#what-it-is}
+
+持續交付，通常縮寫為 CD，是一套實踐，
```
Comment (pichuang) [inline content/zh-tw/continuous-delivery.md:10]:
```suggestion
持續交付，通常縮寫為 CD，是一套將原始碼變更自動部署到驗收環境中的實踐
```

File: content/zh-tw/continuous-delivery.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+其中程式碼的變化被自動部署到驗收環境中
```
Comment (pichuang) [inline content/zh-tw/continuous-delivery.md:11]:
```suggestion
```

File: content/zh-tw/continuous-delivery.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+（或者，在持續部署的情況下，部署到正式環境中）。
+CD 關鍵是包括確定軟體在部署前
+得到充分測試的程式，並提供一種在認為必要時退回修改的方法。
```
Comment (pichuang) [inline content/zh-tw/continuous-delivery.md:14]:
```suggestion
得到充分測試的程式，並提供一種在必要時退回修改的方法。
```

File: content/zh-tw/continuous-delivery.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+持續整合（CI）是實現持續交付的第一步
+（也就是說，在測試和部署之前，變更必須乾淨地合併）。
+
+## 解決的問題 {#problem-it-addresses}
+
+部署 [可靠](/zh-tw/reliability/) 的更新在規模上成為一個問題。
```
Comment (pichuang) [inline content/zh-tw/continuous-delivery.md:20]:
```suggestion
大規模部署 [可靠](/zh-tw/reliability/) 的更新會成為一個問題。
```

File: content/zh-tw/continuous-delivery.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+理想情況下，我們會更頻繁地部署，為終端使用者提供更好的價值。
+然而，手動操作會使每一個變化都轉變為高額的交易成本。
+過去，為了避免這些成本，企業發布的頻率較低，
+一次部署更多的變更，增加了出錯的風險。
+
+## 如何幫助 {#how-it-helps}
+
+CD 策略建立了一個完全自動化的生產路徑，
+使用各種部署策略測試和部署軟體，
+如 [金絲雀部署](/zh-tw/canary-deployment/) 或 [藍綠部署](/zh-tw/blue-green-deployment) 發布。
```
Comment (pichuang) [inline content/zh-tw/continuous-delivery.md:30]:
```suggestion
如 [金絲雀部署](/zh-tw/canary-deployment/) 或 [藍綠部署](/zh-tw/blue-green-deployment) 來進行發布。
```

File: content/zh-tw/continuous-delivery.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+這使得開發人員可以頻繁地部署程式碼，讓他們放心地認為新的修訂版已經過測試。
```
Comment (pichuang) [inline content/zh-tw/continuous-delivery.md:31]:
```suggestion
這使得開發人員可以頻繁地部署程式碼，讓他們放心地確保新的修訂版已經過測試。
```

File: content/zh-tw/continuous-delivery.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+通常情況下，CD 策略中使用基於主分支的開發，而不是功能分支或拉取要求。
```
Comment (pichuang) [inline content/zh-tw/continuous-delivery.md:32]:
```suggestion
通常情況下，CD 策略中使用基於主幹的開發方式，而不是功能分支或拉取要求。
```

**Review by pichuang — COMMENTED** [review]
Check

File: content/zh-tw/continuous-delivery.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+持續整合（CI）是實現持續交付的第一步
+（也就是說，在測試和部署之前，變更必須乾淨地合併）。
+
+## 解決的問題 {#problem-it-addresses}
+
+大規模部署 [可靠](/zh-tw/reliability/) 的更新會成為一個問題。
+理想情況下，我們會更頻繁地部署，為終端使用者提供更好的價值。
+然而，手動操作會使每一個變化都轉變為高額的交易成本。
+過去，為了避免這些成本，企業發布的頻率較低，
+一次部署更多的變更，增加了出錯的風險。
```
Comment (ydFu) [inline content/zh-tw/continuous-delivery.md:23]:
```suggestion
在一次的部署中包含更多的變更，同時也會增加出錯的風險。
```

File: content/zh-tw/continuous-delivery.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+得到充分測試的程式，並提供一種在必要時退回修改的方法。
+持續整合（CI）是實現持續交付的第一步
+（也就是說，在測試和部署之前，變更必須乾淨地合併）。
+
+## 解決的問題 {#problem-it-addresses}
+
+大規模部署 [可靠](/zh-tw/reliability/) 的更新會成為一個問題。
+理想情況下，我們會更頻繁地部署，為終端使用者提供更好的價值。
+然而，手動操作會使每一個變化都轉變為高額的交易成本。
+過去，為了避免這些成本，企業發布的頻率較低，
```
Comment (ydFu) [inline content/zh-tw/continuous-delivery.md:22]:
```suggestion
過去，為了避免上述成本，企業發布的頻率較低，
```

File: content/zh-tw/continuous-delivery.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+持續交付，通常縮寫為 CD，是一套將原始碼變更自動部署到驗收環境中的實踐，
+（或者，在持續部署的情況下，部署到正式環境中）。
+CD 關鍵是包括確定軟體在部署前
+得到充分測試的程式，並提供一種在必要時退回修改的方法。
+持續整合（CI）是實現持續交付的第一步
+（也就是說，在測試和部署之前，變更必須乾淨地合併）。
+
+## 解決的問題 {#problem-it-addresses}
+
+大規模部署 [可靠](/zh-tw/reliability/) 的更新會成為一個問題。
```
Comment (ydFu) [inline content/zh-tw/continuous-delivery.md:19]:
```suggestion
大規模部署具有 [可靠性](/zh-tw/reliability/) 的更新會成為一個問題。
```

File: content/zh-tw/continuous-delivery.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+持續交付，通常縮寫為 CD，是一套將原始碼變更自動部署到驗收環境中的實踐，
+（或者，在持續部署的情況下，部署到正式環境中）。
+CD 關鍵是包括確定軟體在部署前
+得到充分測試的程式，並提供一種在必要時退回修改的方法。
+持續整合（CI）是實現持續交付的第一步
+（也就是說，在測試和部署之前，變更必須乾淨地合併）。
+
+## 解決的問題 {#problem-it-addresses}
+
+大規模部署具有 [可靠](/zh-tw/reliability/) 的更新會成為一個問題。
```
Comment (ydFu) [inline content/zh-tw/continuous-delivery.md:19]:
```suggestion
大規模部署具有 [可靠](/zh-tw/reliability/) 的更新會成為一個問題。
```
reliability:可靠性
ref. form cambridge:
https://dictionary.cambridge.org/zht/%E8%A9%9E%E5%85%B8/%E8%8B%B1%E8%AA%9E-%E6%BC%A2%E8%AA%9E-%E7%B9%81%E9%AB%94/reliability

**Review by ydFu — APPROVED** [review]
LTGM,thank @tico88612

**Review by pichuang — APPROVED** [review]
LGTM

**Pull request merged**